spec/components/concern/category_hashtag_spec.rb

```diff
@@ -29,14 +29,6 @@
       expect(Category.query_from_hashtag_slug("non-existent#{CategoryHashtag::SEPARATOR}#{parent_category.slug}")).to eq(nil)
     end
 
-    it "should be case sensitive" do
```

Comment:
Do we have a test that ensures slugs are case insensitive?

Reply:
There is a `list_controller` spec to test that a redirect to correct slug happens.